browser/ui/brave_ads/ad_notification_popup.cc

```diff
@@ -422,7 +427,15 @@ void AdNotificationPopup::CreateWidgetView() {
   params.type = views::Widget::InitParams::TYPE_WINDOW_FRAMELESS;
   params.z_order = ui::ZOrderLevel::kFloatingWindow;
   params.opacity = views::Widget::InitParams::WindowOpacity::kTranslucent;
+#if defined(OS_LINUX)
+  // Some linux configurations do not support transparent content drawing. This
+  // can cause artifacts on shadows around ads notification popup.
+  // To fix this shadows are drawn by Widget.
+  params.shadow_type = views::Widget::InitParams::ShadowType::kDrop;
```

Comment:
I had rendering issues on Windows and macOS when the shadow was drawn by the Widget, any ideas if your change will work on macOS and Windows?

Reply:
What kind of issues were drawn? I will check it on Windows.

Reply:
The shadow was not always rendered until dragging the widget to another screen location

Reply:
I didn't noticed such rendering issues on Windows. 
Shadow is drawn but InitParams::shadow_elevation and InitParams::corner_radius parameters are ignored.

Reply:
Once ready for review I will retest on Windows 10 and macOS. Thanks